### ISSUE TITLE
BUILD Error on modules/face_geometry/geometry_pipeline_calculator

### ISSUE THREAD

**Issue by PrinceRay7** [google-ai-edge/mediapipe#5704]
### OS Platform and Distribution

Linux Ubuntu 22.04

### Compiler version

gcc/g++ 11.4.0

### Programming Language and version

C++

### MediaPipe version

0.10.18 (latest)

### Bazel version

6.1.1

### Android AAR (if android)

None

### OpenCV version (if running on desktop)

4.5

### Describe the problem

Build Error on geometry_pipeline_calculator and effect_render_calculator: error: cannot convert 'absl::string_view'

### Complete Logs

```shell
When build with the command:
'bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=0 //mediapipe/modules/face_geometry:geometry_pipeline_calculator'
ERROR: cannot convert 'absl::string_view' {aka 'std::basic_string_view<char>'} to 'google::protobuf::ConstStringParam' {aka 'const std::__cxx11::basic_string<char>&'}

The Full log is here:

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
In file included from external/com_google_absl/absl/base/macros.h:36,
                 from external/com_google_absl/absl/memory/memory.h:33,
                 from mediapipe/modules/face_geometry/geometry_pipeline_calculator.cc:20:
mediapipe/modules/face_geometry/geometry_pipeline_calculator.cc: In static member function 'static absl::StatusOr<mediapipe::face_geometry::GeometryPipelineMetadata> mediapipe::{anonymous}::GeometryPipelineCalculator::ReadMetadataFromFile(mediapipe::CalculatorContext*, const string&)':
mediapipe/modules/face_geometry/geometry_pipeline_calculator.cc:172:67: error: cannot convert 'absl::string_view' {aka 'std::basic_string_view<char>'} to 'google::protobuf::ConstStringParam' {aka 'const std::__cxx11::basic_string<char>&'}
  172 |     RET_CHECK(metadata.ParseFromString(metadata_blob->ToStringView()))
      |                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
      |                                                                   |
      |                                                                   absl::string_view {aka std::basic_string_view<char>}
external/com_google_absl/absl/base/optimization.h:178:59: note: in definition of macro 'ABSL_PREDICT_FALSE'
  178 | #define ABSL_PREDICT_FALSE(x) (__builtin_expect(false || (x), false))
      |                                                           ^
mediapipe/modules/face_geometry/geometry_pipeline_calculator.cc:172:5: note: in expansion of macro 'RET_CHECK'
  172 |     RET_CHECK(metadata.ParseFromString(metadata_blob->ToStringView()))
      |     ^~~~~~~~~
In file included from external/com_google_protobuf/src/google/protobuf/generated_enum_util.h:36,
                 from external/com_google_protobuf/src/google/protobuf/map.h:59,
                 from external/com_google_protobuf/src/google/protobuf/generated_message_table_driven.h:34,
                 from bazel-out/k8-opt/bin/mediapipe/framework/calculator.pb.h:26,
                 from ./mediapipe/framework/calculator_state.h:27,
                 from ./mediapipe/framework/calculator_context.h:25,
                 from ./mediapipe/framework/calculator_base.h:26,
                 from ./mediapipe/framework/calculator_framework.h:54,
                 from mediapipe/modules/face_geometry/geometry_pipeline_calculator.cc:23:
external/com_google_protobuf/src/google/protobuf/message_lite.h:285:74: note:   initializing argument 1 of 'bool google::protobuf::MessageLite::ParseFromString(google::protobuf::ConstStringParam)'
  285 |   PROTOBUF_ATTRIBUTE_REINITIALIZES bool ParseFromString(ConstStringParam data);
      |                                                         ~~~~~~~~~~~~~~~~~^~~~
Target //mediapipe/modules/face_geometry:face_geometry_from_landmarks failed to build
INFO: Elapsed time: 14.091s, Critical Path: 13.60s
INFO: 7 processes: 2 internal, 5 processwrapper-sandbox.
FAILED: Build did NOT complete successfully


I checked the error, which seems like a string converter error. It will occur in the file <effect_renderer_calculator.cc> & <geometry_pipeline_calculator.cc> at the line 

'RET_CHECK(metadata.ParseFromString(metadata_blob->ToStringView()))
     << "Failed to parse a metadata proto from a binary blob!";'

basically it has some problem on converting the metadata_blob into strings. So the question is how to solve this? 
It surprised me that none of any issue has mentioned this problem, nobody meets this before? 
I've successfully built other demos like face mesh and object detection, however, in the case of rendering, I couldn't go farther. 
Did I miss any flags when running bazel build? Could somebody help me on this error?
 I've done a lot of search and no right answers so far. If somebody has successfully built the face_geometry module, 
please tell me how, and I really appreciate it.
```

**Comment by PrinceRay7** [comment]
I've also noticed that in effect_renderer_calculator.cc,  this line also presents the error **error: cannot convert 'absl::string_view' {aka 'std::basic_string_view<char>'} to 'google::protobuf::ConstStringParam' {aka 'const std::__cxx11::basic_string<char>&'}**

RET_CHECK(mesh_3d.ParseFromString(mesh_3d_blob->ToStringView()))

I noticed that in the BUILD file of face_geometry, under the cc_library of effect_render_calculator, it doesn't has the dependency         "@com_google_absl//absl/strings:string_view", however, adding this dep couldn't solve this problem.

**Comment by PrinceRay7** [comment]
@kuaashish Sorry to bother, I really appreciate it if you could take a look at this, thank you so much!

**Comment by kuaashish** [comment]
Hi @PrinceRay7,

This was known issue, and recent in this commit https://github.com/google-ai-edge/mediapipe/commit/3a79fffc363ebe4e19f73d49eb255e0ddee1d103, We have fixed the issue, Can you please check and build the command, Now there should be no error we should receive. 

Thank you!!

**Comment by PrinceRay7** [comment]
thank you for your answer, the build has been successful

**Comment by kuaashish** [comment]
Hi @PrinceRay7,

Thank you for confirming. May we mark this issue as resolved and close it?
